zoomus/client.py

```diff
@@ -26,6 +26,7 @@
         "report": components.report.ReportComponentV2,
         "webinar": components.webinar.WebinarComponentV2,
         "recording": components.recording.RecordingComponentV2,
+        "live_stream": components.live_stream.LiveStreamComponentV2,
```

Comment:
I'm not sure if I made this list alphabetical in develop, but can I trouble you to order this list here and ensure that it is in alphabetical order? Thanks!

Reply:
I think this is already changed